### PR TITLE
Allow Padatious to override Adapt

### PR DESCRIPTION
**Pull request by penrods**
Allow a Padatious intent to override Adapt when it is VERY
certain that the utterance is directed at it.  (95% confidence
or greater.)  Right now that only occurs if the intent match
for the given phrase is perfect.

This solves this kind of issue:
* Adapt:  Matching on "Set" and "Alarm"
* Padatious: Handling "is an alarm set"

## Description
(Description of what the PR does, such as fixes # {issue number})

## How to test
Can be tested by creating the example mentioned above:
* Adapt:  Matching on "Set.voc" and "Alarm.voc" (containing those words)
* Padatious: Handling "alarm.status" that contains "is an alarm set"

The Padatious intent should match on "is an alarm set" but not on "is a alarm set"

## Contributor license agreement signed?
CLA [ X ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

File: mycroft/skills/intent_service.py [hunk 1]
```diff
@@ -287,15 +287,18 @@ def handle_utterance(self, message):
...
 
             if converse:
                 # Report that converse handled the intent and return
                 ident = message.context['ident'] if message.context else None
                 report_timing(ident, 'intent_service', stopwatch,
                               {'intent_type': 'converse'})
                 return
-            elif intent:
-                # Send the message to the intent handler
+            elif intent and padatious_intent and padatious_intent.conf < 0.95:
```
Comment (MatthewScholefield) [inline mycroft/skills/intent_service.py]: What if there's no Padatious intent but there is an adapt intent? (Logic should probably be `if intent and not (padatious_intent and padatious_intent.conf < 0.95):`)

Comment (penrods) [reply mycroft/skills/intent_service.py]: Ah, yup!  Changing.

Comment (penrods) [reply mycroft/skills/intent_service.py]: Except the confidence level comparison needs to be inverted, too:
```python
elif intent and not (padatious_intent and padatious_intent.conf >= 0.95):
```

**Comment by penrods** [comment]
Yep, definitely a hack at the moment!

**Pull request merged**